src/thor/dijkstras.cc

```diff
@@ -693,7 +624,8 @@ void Dijkstras::ExpandForwardMultiModal(GraphReader& graphreader,
       continue;
     }
 
-    // Make the label in advance, we may not end up using it but we need it for the expansion decision
+    // Make the label in advance, we may not end up using it but we need it for the expansion
+    // decision
```

Comment:
nit: saw this above as well, does the linter have different widths in different peoples editors?

Reply:
Fixed this - it was because I moved the code into the `.h` file initially that had a slightly different indentation depth, then moved it back.